### PR TITLE
Scraper updates

### PR DESCRIPTION
**Pull request by rwalek668**
Updating the scraper to improve the readability of the code, and to have the scraper jump to pages rather then clicking "next" through every page.  On rare occasions, the scraper will still sometimes error out if a specific page takes too long to load.

Feel free to be critical on this review including style/readability of the code.

### CODE REVIEW

File: data_pipeline/scraper/scraper.py [hunk 1]
```diff
@@ -19,65 +19,101 @@
...
+            EC.presence_of_element_located((By.ID, 'ctl00_DefaultContent_ASPxRoundPanel1_btnFindFilers_CD')))
 
-                # TODO(walek): Update this to use contains
-                if self.driver.find_elements_by_xpath('//*[@id="ctl00_GridContent_popupCantContinueDialog_Button1"]'):
-                    self.driver.find_elements_by_xpath('//*[@id="ctl00_GridContent_popupCantContinueDialog_Button1"]')[0].click()
-                    error_dialog = True
-                    continue
+        # Search, which will load up the content on the page.
+        # Search terms are left blank, indicating "all content".
+        driver.find_element_by_xpath('//*[@id="ctl00_DefaultContent_ASPxRoundPanel1_btnFindFilers_CD"]').click()
```
Comment (JMStudiosJoe) [inline data_pipeline/scraper/scraper.py:40]: can we set these ids to better var names that are human readable and make more sense? helps navigate the scrapper even in code

Comment (JMStudiosJoe) [reply data_pipeline/scraper/scraper.py:40]: same with any others

File: data_pipeline/scraper/scraper.py [hunk 2]
```diff
@@ -93,47 +129,37 @@ def __initDrivers(self):
                  "plugins.plugins_list": [plugs]}
         options.add_experimental_option("prefs", prefs)
         self.driver = webdriver.Chrome(ChromeDriverManager().install(), options=options)
-
         # bypass chromedriver headless security
```
Comment (JMStudiosJoe) [inline data_pipeline/scraper/scraper.py:132]: can we remove the commented out code?

**Comment by JMStudiosJoe** [comment]
Great so far just a couple cleanup and question or two.

File: data_pipeline/scraper/scraper.py (hunk 2 above)
Comment (rwalek668) [reply data_pipeline/scraper/scraper.py]: Good point, updated.

File: data_pipeline/scraper/scraper.py (hunk 1 above)
Comment (rwalek668) [reply data_pipeline/scraper/scraper.py]: The ID's are generated as part of the city website which unfortunately we do not control.  I did abstract these out into variables though to try to make it more readable at least within this file. Thanks for this input.

**Pull request merged**